### PR TITLE
Internationalization: Fixes

### PR DESCRIPTION
**Pull request by jniederm**
... based on communication with Yuko:

* Awkward English reworded.
* Sentences split to multiple parts because of intermediate HTML tags
  replaced by whole sentences including HTML marking because translators
  struggled with context (even when additional description was
  provided).

### CODE REVIEW

File: src/components/OvirtApiCheckFailed.js [hunk 1]
```diff
@@ -27,8 +27,7 @@ const OvirtApiCheckFailed = ({ config }) => {
   return (
     <div className='alert alert-danger'>
       <span className='pficon pficon-error-circle-o' />
-      <strong>{msg.unsupportedOvirtVersionFound({ version: found, productName: fixedStrings.BRAND_NAME })}</strong>,&nbsp;
-      {msg.butVersionAtLeastRequired({ requiredVersion: required })}.
+      <span dangerouslySetInnerHTML={msg.htmlUnsupportedOvirtVersionFoundButVersionAtLeastRequired({ version: found, productName: fixedStrings.BRAND_NAME, requiredVersion: required })}></span>
```
Comment (mareklibra) [inline src/components/OvirtApiCheckFailed.js]: Please use self-closing <span /> to make linter happy

File: src/intl/messages.js [hunk 2]
```diff
@@ -73,18 +69,7 @@ export const messages = {
...
-  },
-  pleaseReferToDocumentationForMoreInformationPart2: {
-    message: 'documentation',
-    description: 'Part 2/3 of sentence: Please refer to | documentation | for more information.',
-  },
-  pleaseReferToDocumentationForMoreInformationPart3: {
-    message: 'for more information',
-    description: 'Part 3/3 of sentence: Please refer to | documentation | for more information.',
-  },
+  htmlPleaseReferToDocumentationForMoreInformation: 'Please refer to <a href="{documentationUrl}" target="_blank">documentation</a> for more information.',
```
Comment (mareklibra) [inline src/intl/messages.js]: @jniederm , have you verified that this inner `<a>` works as expected, please?

File: src/components/OvirtApiCheckFailed.js (hunk 1 above)
Comment (jniederm) [inline src/components/OvirtApiCheckFailed.js]: done

File: src/intl/messages.js (hunk 2 above)
Comment (jniederm) [inline src/intl/messages.js]: now it works, verified

**Comment by mareklibra** [comment]
@jniederm , have you force-pushed latest changes? :-)

**Comment by jniederm** [comment]
Thanks for reminding me. Changes pushed.

**Pull request merged**